solidity/app/cl_utils.js

```diff
@@ -0,0 +1,33 @@
+const Eth = require('ethjs')
+
+
+clUtils = {
+  personalAccount: '0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f',
+  toWei: function toWei (eth) {
+    return (parseInt(eth.toString()) * 10 ** 18).toString()
+  },
+  getTxReceipt: function getTxReceipt (txHash) {
+    return new Promise(async (resolve, reject) => {
+      for (let i = 0; i < 1000; i++) {
```

Comment:
What is `1000`?

Reply:
Magic number. I wanted it to retry a lot, but not forever. Probably worth making configurable.